Modules/ArcGIS/DSCResources/ArcGIS_Server_TLS/ArcGIS_Server_TLS.psm1

```diff
@@ -1022,12 +1022,12 @@ function Get-SSLCertificateForMachine
                 $null 
             }else {
                 $IssuerValue = $null
-                $Issuer = [regex]::matches($res.Content, '"issuer":"([A-Za-z =,\.0-9\-]+)\"')
+                $Issuer = [regex]::matches($res.Content, '"Issuer":[ ]?\"([A-Za-z =,\.0-9\-]+)\"')
                 $Issuer.Groups | %{ 
                     if($_.Value -and $_.Value.Length -gt 0){
-                        $Pos = $_.Value.IndexOf('"issuer"')
+                        $Pos = $_.Value.IndexOf('"Issuer"')
```

Comment:
On our site `issuer` is all lowercase. Maybe the check could be case-insensitive?